src/zope/sequencesort/tests/test_ssort.py

```diff
@@ -297,9 +297,17 @@ def _makeField(self, name, _cmp=None, multiplier=1):
         if _cmp is None:
             try:
                 _cmp = cmp
-            except NameError: #pragma NO COVER Py3k
+            except NameError:
                 def _cmp(lhs, rhs):
-                    return int(rhs < lhs) - int(lhs < rhs)
```

Comment:
Unfortunately, because of this definition, this isn't actually testing the code in `ssort.py`. I verified this by reverting the changes in `ssort.py` and observing that the new `test_none` still passes. 

I suggest changing this function to be simply:
```python
if _cmp is None:
    from zope.sequencesort.ssort import cmp as _cmp
```

That way, we test the actual comparison function.